cmake/cmake_test/add_dir.cmake

```
@@ -168,5 +176,10 @@ function(ct_add_dir _ad_test_dir)
                 -B "${_ad_test_dest_full_path}"
                 ${ADD_DIR_CMAKE_OPTIONS}
         )
+        if(ADD_DIR_LABEL AND NOT ADD_DIR_LABEL_NAME)
+            set_property(TEST ${_ad_test_name} PROPERTY LABELS ${_ad_test_dir})
+        elseif(ADD_DIR_LABEL_NAME)
+            set_property(TEST ${_ad_test_name} PROPERTY LABELS "${ADD_DIR_LABEL_NAME}")
+        endif()
```

Switch these cases for easier logic while `LABEL_NAME` still takes precedence over `LABEL`:

```suggestion
        if(ADD_DIR_LABEL_NAME)
            set_property(TEST "${_ad_test_name}" PROPERTY LABELS "${ADD_DIR_LABEL_NAME}")
        elseif(ADD_DIR_LABEL)
            set_property(TEST "${_ad_test_name}" PROPERTY LABELS "${_ad_test_dir}")
        endif()
```

---

In my experience, it is usually a good idea to put double quotes around all standalone variable accesses unless you have an explicit reason not to, like if you intend to expand a list or access a variable reference. For consistency I added them, although I don't think it makes a difference here.